src/GafferImageUI/ImageGadget.cpp

```diff
@@ -110,6 +110,8 @@ void findUsableTextureFormats( GLenum &monochromeFormat, GLenum &colorFormat )
 	monochromeFormat = g_monochromeFormat;
 	colorFormat = g_colorFormat;
 }
+
+std::atomic<uint64_t> g_tileUpdateCount;
```

Comment:
I don't think this needs to be atomic - it's only accessed from `Tile::texture()`, which has to run on the main thread.

Reply:
Removed atomic.